presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -368,6 +366,10 @@ private static boolean shouldUseFileSplitsFromInputFormat(InputFormat<?, ?> inpu
         return splitList;
     }
 
+    private String getCorruptBucketingMsg() {
+        return "Hive table " + table.getDatabaseName() + "." + table.getTableName() + " is corrupt. It is declared as being bucketed, but the files do not match the bucketing declaration.";
```

Comment:
Use `format()` as it's easier to read. Also add single quotes around the table name in the error message. See `TableNotFoundException`

Reply:
ok changed to SchemaTableName